src/howTos/dev/services.md

```diff
@@ -104,6 +104,14 @@ node "${arg}" | tee -a ~/.cozy/services.log
 
 Now you can `tail -f ~/.cozy/services.log` to watch logs in real time.
 
+### Get all errors and debug messages
```

Comment:
Instead of adding another paragraph, I would have completed the preceding one.

Reply:
@sblaisot better?

Reply:
:+1: